server/src/test/java/io/crate/planner/operators/FetchRewriteTest.java

```diff
@@ -52,6 +56,16 @@
 
 public class FetchRewriteTest extends CrateDummyClusterServiceUnitTest {
 
+    public static Function of(String name, List<Symbol> arguments, DataType<?> returnType) {
+        return new Function(
```

Comment:
In some tests, you don't use signatures to create function instances. e.g. `JoinPhaseTest`, `FilterProjectionTest`, etc. is it by intention or it is just slipped from the sight?

Reply:
Somehow by intention, in these tests the function registry is not used so it doesn't really matter. This will change anyway when the `signature()` method is required to return non-null values, I'm already working on this a part of removing the old function registry.